internal/system/agent/execution_engine.go

```diff
@@ -23,7 +24,27 @@ func runExec(service string, operation string) error {
 	return err
 }
 
-func (ec *ExecuteApp) Start(service string) error {
+func execMetrics(service string) ([]byte, error) {
+
+	// Preparing the call to the executor app.
+	//var result []byte
+
+	cmd := exec.Command(Configuration.ExecutorPath, service, METRICS)
+
+	out, err := cmd.CombinedOutput()
+	LoggingClient.Info(fmt.Sprintf("err.Error(): %s ", err.Error()))
+	if err.Error() != "exit status 1" {
```

Comment:
The logic here looks wrong to me. How this is written if err is `exit status 1`, the SMA will think it succeeded which seems wrong. Shouldn't `exit status 1` still be considered an error?